src/common/libsubprocess/subprocess.h

```diff
@@ -104,6 +109,8 @@ typedef struct {
     flux_subprocess_output_f on_channel_out; /* Read from channel when ready */
     flux_subprocess_output_f on_stdout; /* Read of stdout is ready           */
     flux_subprocess_output_f on_stderr; /* Read of stderr is ready           */
+    flux_subprocess_credit_f on_credit; /* bytes of write buffer space
+                                         * reclaimed */
```

Comment:
Maybe just say `// Write buffer space available` for the inline comment, since this is also called for initial credits, where there is no reclamation.